wgpu-hal/src/dx11/device.rs

```diff
@@ -227,12 +227,14 @@ impl crate::Queue<super::Api> for super::Queue {
 impl super::D3D11Device {
     #[allow(trivial_casts)] // come on
     pub unsafe fn check_feature_support<T>(&self, feature: d3d11::D3D11_FEATURE) -> T {
-        let mut value = mem::zeroed::<T>();
-        let ret = self.CheckFeatureSupport(
-            feature,
-            &mut value as *mut T as *mut c_void,
-            mem::size_of::<T>() as u32,
-        );
+        let mut value = unsafe { mem::zeroed::<T>() };
```

Comment:
This kind of function barely benefits - we could just make it one large unsafe

Reply:
Changed in 5a2d2c5b.